medicines/doc-index-updater/src/main.rs

```diff
@@ -10,15 +10,12 @@ const PORT: u16 = 8000;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn error::Error>> {
-    let subscriber = tracing_subscriber::fmt::Subscriber::builder()
-        .json()
-        .flatten_event(true)
-        .with_timer(tracing_subscriber::fmt::time::ChronoUtc::rfc3339())
-        .with_max_level(Level::INFO)
-        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
-        .finish();
+    if get_env_or_default("JSON_LOGS", true) {
+        use_json_log_subscriber()
+    } else {
+        use_unstructured_log_subscriber()
```

Comment:
Small suggestion - might make unit testing easier (when we get around to that) if these functions returned a subscriber that you then `set_global_default` with on the next line.

Reply:
the only reason we didn't do that is because the .json function then changes the generic type slightly